project-leadership.md

```diff
@@ -29,6 +30,8 @@ One example of success in this area: You are anchoring your team. You’ve been
   <li>Volunteers for facilitating team meetings</li>
 
   <li>Discusses team “things” with their anchor (eg: team process, team health/morale, product, architecture)</li>
+
+  <li>Is comfortable letting go of a priority that they have worked on</li>
```

Comment:
There might be a more concrete way of expressing this, or perhaps color to be added:
> Is comfortable when a thing they have worked on is no longer a priority

... maybe this idea is better as an "explainer" for this?

Reply:
Actually, we're just going to drop this one.  It is the absence of a reaction, and isn't really worth trying to track.